sdk/servicebus/service-bus/src/session/sessionManager.ts

```diff
@@ -153,7 +100,7 @@ export class SessionManager {
           this._maxPendingAcceptSessionsSemaphore.awaitedTaskCount()
         );
 
-        const closeMessageSession = async (messageSession: MessageSession) => {
+        const closeMessageSession = async (messageSession: MessageSession): Promise<void> => {
```

Comment:
@ramya-rao-a  what would be the appropriate type for `closeMessageSession`  ?

Reply:
Promise<void> is correct